src/LineChart.ts

```diff
@@ -34,6 +34,7 @@ module n3Charts {
         ['container', Factory.Container, element[0]],
         ['x-axis', Factory.Axis, 'x'],
         ['y-axis', Factory.Axis, 'y'],
+        ['lines-container', Factory.Series.LineSeriesContainer, 'y'],
```

Comment:
Ok I have the same thing in mind. I will adapt this later and propose a great and simple solution